ui/src/style/unsorted.scss

```diff
@@ -90,3 +90,31 @@
     margin-bottom: 11px;
   }
 }
+
+/*
+    Loading Dots
+    ----------------------------------------------
+*/
+
+.loading-dots {
+  position: absolute;
+  transform: translate(0,0);
+  transform: translateX(50%);
+  width: 16px;
+  height: 18px;
+
+  > div {
+    width: 4px;
+    height: 4px;
+    background-color: $g6-smoke;
+    border-radius: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%,-50%);
+  }
+
+  div:nth-child(1) {left: 0; animation: refreshingSpinnerA 0.8s cubic-bezier(0.645, 0.045, 0.355, 1) infinite;}
+  div:nth-child(2) {left: 50%; animation: refreshingSpinnerB 0.8s cubic-bezier(0.645, 0.045, 0.355, 1) infinite;}
+  div:nth-child(3) {left: 100%; animation: refreshingSpinnerC 0.8s cubic-bezier(0.645, 0.045, 0.355, 1) infinite;}
+}
```

Comment:
@alexpaxton feel free to lay down the css law if you dislike ^. I also noticed that `refreshingSpinnerA` - `C` apply a constant `translate(-50%, -50%)` - how come?